setup.py

```diff
@@ -1,52 +0,0 @@
-print("This application relies on Stripe API keys.")
```

Comment:
The setup instructions say repeatedly that you must run `setup.py`. Now it's been removed.

Reply:
This was already merged. I plan on updating the readme in a new PR after finishing up the swagger docs.

The new process will be to copy `.env.dist` to `.env` and update any values that need to be updated.